src/components/CheckoutDialog.jsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogClose,
+  DialogTrigger,
+  DialogContent,
+  Button,
+  Flex,
+} from "@sparrowengg/twigs-react";
+import ShippingAddress from "./ShippingAddress";
+import OrderSummary from "./OrderSummary";
+export const CheckoutDialog = () => {
+  return (
+    <>
```

Comment:
@arun-121 is this empty fragment needed???

Reply:
removed the same